cmd/vcpctl/provision/provision.go

```diff
@@ -76,7 +76,7 @@ func AddProvision(cmd *cobra.Command) {
 	provisionCmd.Flags().StringVar(&vcport, "port", "", "Specify vCenter Port")
 	provisionCmd.Flags().StringVar(&vcUser, "user", "", "Specify vCenter user")
 	provisionCmd.Flags().StringVar(&vcPassword, "password", "", "Specify vCenter Password")
-	provisionCmd.Flags().BoolVar(&insecure, "insecure", false, "Don't verify the server's certificate chain")
+	provisionCmd.Flags().BoolVar(&insecure, "insecure", true, "Don't verify the server's certificate chain")
```

Comment:
isn't this setting insecure `true` by default?

Reply:
Done. Thanks @fabianofranz @akutz